udata_front/theme/gouvfr/assets/js/pages/ReusePublishingForm/Step1DescribeReuse.vue

```diff
@@ -0,0 +1,397 @@
+<template>
+  <div class="fr-container">
+    <Stepper :steps="steps" :currentStep="0"/>
+    <div class="fr-grid-row">
+      <Sidemenu
+        class="fr-col-12 fr-col-md-5"
+        :buttonText="t('Help')"
+        :onRight="true"
+        :fixed="true"
+        >
+          <template #title>
+            <span class="fr-icon--sm fr-icon-question-line" aria-hidden="true"></span>
+            {{ t('Help') }}
+          </template>
+          <AccordionGroup>
+            <Accordion
+              :title= "t('Naming your reuse')"
+              :id="nameReuseAccordionId"
+              :state="state.title"
+            >
+              <p class="fr-m-0">
+                {{ t("Prefer a title that allows understanding of the use made of the data rather than the name of the website or application (\"Search Engine for Company Agreements\" rather than \"\" for example).") }}
```

Comment:
```suggestion
                {{ t('Prefer a title that allows understanding of the use made of the data rather than the name of the website or application ("Search Engine for Company Agreements" rather than "Accords-entreprise.fr" for example).') }}
```